specs/IFrameMemoryUsage

```diff
@@ -0,0 +1,251 @@
+  /// Provides the result for the process changed event.
+  /// No event args exist and the args parameter is set to null.
+  HRESULT Invoke([in] ICoreWebView2 * sender,
+                 [in] IUnknown* args);
+}
+
+/// Receives `ProcessChanged` event.
+// MSOWNERS: wangsongjin@microsoft.com
+[uuid(c1e0a3d2-a74b-11ed-afa1-0242ac120002), object, pointer_default(unique)]
+interface ICoreWebView2FrameRenderProcessChangedEventHandler : IUnknown {
+  /// Provides the result for the process changed event.
+  HRESULT Invoke([in] ICoreWebView2Frame* sender,
+                 [in] IUnknown* args);
+}
+
+/// This is an extension of the ICoreWebView2Frame interface.
+// MSOWNERS: wangsongjin@microsoft.com
+[uuid(04baa798-a0e9-11ed-a8fc-0242ac120002), object, pointer_default(unique)]
+interface ICoreWebView2Frame5 : ICoreWebView2Frame4 {
+  /// Raised when a new iframe is created.
+  /// Handle this event to get access to ICoreWebView2Frame objects.
+  /// Use ICoreWebView2Frame.add_Destroyed to listen for when this iframe goes
+  /// away.
+  // MSOWNERS: wangsongjin@microsoft.com
+  HRESULT add_FrameCreated(
+      [in] ICoreWebView2NestedFrameCreatedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Remove an event handler previously added with add_FrameCreated.
+  // MSOWNERS: wangsongjin@microsoft.com
+  HRESULT remove_FrameCreated([in] EventRegistrationToken token);
+
+  /// Get the render process id of the iframe
+  // MSOWNERS: wangsongjin@microsoft.com
+  [propget] HRESULT ProcessId([out, retval] UINT32* value);
+
+  /// Subscribe to render process Id changed event
+  // MSOWNERS: wangsongjin@microsoft.com
+  HRESULT add_ProcessIdChanged(
+      [in] ICoreWebView2FrameRenderProcessChangedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Remove an event handler previously added with add_ProcessIdChanged.
+  // MSOWNERS: wangsongjin@microsoft.com
+  HRESULT remove_ProcessIdChanged([in] EventRegistrationToken token);
+}
+
+/// A continuation of the `ICoreWebView2` interface to support get 
+/// render process id and handle render process change event
+[uuid(ad712504-a66d-11ed-afa1-0242ac120002), object, pointer_default(unique)]
+interface ICoreWebView2_18 : ICoreWebView2_17 {
+  /// Get the render process id of the iframe
+  // MSOWNERS: wangsongjin@microsoft.com
+  [propget] HRESULT ProcessId([out, retval] UINT32* value);
+
+  /// Subscribe to render process id changed event
+  // MSOWNERS: wangsongjin@microsoft.com
+  HRESULT add_ProcessIdChanged(
+      [in] ICoreWebView2RenderProcessChangedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Remove an event handler previously added with add_ProcessIdChanged.
+  // MSOWNERS: wangsongjin@microsoft.com
+  HRESULT remove_ProcessIdChanged([in] EventRegistrationToken token);
+}
+```
+
+C#
+```c# (but really MIDL3)
+namespace Microsoft.Web.WebView2.Core
+{   
+    runtimeclass CoreWebView2FrameCreatedEventArgs

```

Comment:
This is the same runtimeclass that already exists correct? No modifications? If so and you want to include it please add a comment noting that this is an existing unchanged class.

Reply:
Yep. Will remove this from spec.